src/select-schema.js

```diff
@@ -0,0 +1,6 @@
+define(['$'], function ($) {
+
+    return {
+        'select': $("<select><option>Value</option><option>Value</option></select>")
+    };
```

Comment:
I asked @tedtate about the data schema because I was not really clear on what its role is intended to be. Ted confirmed that the schema is envisioned as “a description of the data required by a component’s Dust template”. It should not contain mock data, which is what we have in select-schema.js. 

Sounds like we’re considering using https://github.com/molnarg/js-schema/ for validation, so following the docs there when authoring our initial components is probably a good idea but I don’t think we really know how this should work yet.